test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -414,28 +434,56 @@ class test_uct_cm_sockaddr : public uct_test {
                        user_data);
     }
 
-    static void
-    cm_conn_request_cb(uct_listener_h listener, void *arg,
-                       const char *local_dev_name,
-                       uct_conn_request_h conn_request,
-                       const uct_cm_remote_data_t *remote_data) {
+
+    /*
+     * Common section for the server's handling of a connection request.
+     * Process the connection request and check if the server's accept is
+     * required for the calling test.
+     *
+     * return true if the server should accept the connection request and
```

Comment:
What about to rename it to `common_conn_request_handler` and invert return value?
`true` - request was handled (rejected in this case)
`false` - request was not handled and caller has to handle calling accept (or reject)

Reply:
i'll rename the function like you suggested.
about the return order - since server_accept is done differently in the classes, i think it's the (only) one that should be outside the common code. i think it's simpler this way..

Reply:
how does it make sense for return code? IMHO err code and funnc name should correlate. 
But since it's local, just for testing and documented, it's not a blocker.